File: libs/common/models/vw-pmp-lookup.ts



export class VwPmpLookup {
    public code: string = '';
    public type: string = '';
    public name: string = '';
    public displayOrder: number = 0;
    public value: string = '';

    constructor(init? : Partial<VwPmpLookup>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/EntityFramework/VwPmpLookup.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models.EntityFramework
{
    public partial class VwPmpLookup
    {
        public string Code { get; set; }
        public string Type { get; set; }
        public string Name { get; set; }
        public int DisplayOrder { get; set; }
        public string Value { get; set; }
    }
}
